lib/cloudregister/smt.py

```diff
@@ -17,7 +17,9 @@
 import ipaddress
 import logging
 import requests
+import os
 
+from urllib.parse import urlparse
```

Comment:
Not really a need for another dependency even if the dependency is part of CPyton standard library. We know all the information this gives us already.

Reply:
Good catch